### ISSUE TITLE
Can not  parse metadata by providing the metadata file.

### ISSUE THREAD

**Issue by thy-chan** [Jialab-UCR/GDCRNATools#2]
I'm trying to use 'gdcParseMetadata()'  to parse metadata by providing the metadata file, but find that the formate of the metadata file in GDC has been change recently.
So, I want to know how long the GDCRNATools can be updated.
Thanks!

`
##############################################################################

Error in vapply(seq_len(nSam), function(i) metadata[[i]]$cases[[1]]$samples[[1]]$submitter_id,  : 
  值的长度必需为1，
 但FUN(X[[1]])结果的长度却是0
Calls: source ... withVisible -> eval -> eval -> gdcParseMetadata -> vapply
停止执行
`

**Comment by Jialab-UCR** [comment]
@thy-chan Hi, could you please tell me the project you are trying to parse? Thanks

**Comment by thy-chan** [comment]
RNA-Seq files in TCGA-BRCA.  The format of the metadata file  has changed so that the GDCRNATools can't parse it. ![image](https://images-cdn.shimo.im/eFApYoGuVSM4IDlA/image.undefined!thumbnail)

**Comment by Jialab-UCR** [comment]
@thy-chan Thank you so much for the reminder. I have just checked the JSON file in GDC and they do have changed the format. There's very little information in the file now, so I recommend that you can parse metadata use the automatic mode (eg., meta <- gdcParseMeta(project.id='TCGA-BRCA', data.type='RNAseq'), and then select the sample you'd like to use. Thanks

**Comment by thy-chan** [comment]
Yes, the information in the GDC's JSON file is reduced. I think the GDC extracts other information separately and puts it in other files, such as the Biospecimen and Sample sheet in the screenshot. Thanks

**Comment by brcangs** [comment]
Parsing the metadata using the automatic mode for the TCGA-BRCA project does unfortunately not seem to work either:

`> metaMatrix <- gdcParseMetadata(project.id='TCGA-BRCA', data.type='RNAseq')`
Error in vapply(seq_len(nSam), function(i) null2naFun(metadata[[i]]$cases[[1]]$demographic$gender),  : 
  values must be type 'character',
 but FUN(X[[297]]) result is type 'logical'

**Comment by ChengYunazhi** [comment]
I got a same error when parsing TCGA-COAD metadata, here it is:
![qq 20180823170957](https://user-images.githubusercontent.com/13846735/44516506-d7a75200-a6f7-11e8-9f5b-dc0b8a879c1b.png)

the command used:
![qq 20180823171348](https://user-images.githubusercontent.com/13846735/44516539-ee4da900-a6f7-11e8-85bc-e6e89bb5d0ad.png)

**Comment by yinlisssss** [comment]
HI, is this because of my internet problem?
or?
thank you very much

> metaMatrix.MIR <- gdcParseMetadata(project.id = "TCGA-LUSC",
+                                    data.type  = "miRNAs")
Error in file(con, "r") : cannot open the connection
In addition: Warning message:
In file(con, "r") : InternetOpenUrl failed: '操作超时'
